packages/flutter/test/material/icon_button_test.dart

```diff
@@ -337,6 +337,40 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('IconButton with explicit min splash radius',
+      (WidgetTester tester) async {
+    const double minSplashRadius = 10.0;
+
+    Widget buttonWidget = wrap(
+      child: IconButton(
+        icon: const Icon(Icons.android),
+        minSplashRadius: minSplashRadius,
+        onPressed: () {/* enable the button */},
+      ),
+    );
+
+    await tester.pumpWidget(
+      Theme(
+        data: ThemeData(),
+        child: buttonWidget,
+      ),
+    );
+
+    final Offset center = tester.getCenter(find.byType(IconButton));
+    final TestGesture gesture = await tester.startGesture(center);
+    await tester.pump(); // start gesture
+    await tester.pump(const Duration(
+        milliseconds: 1000)); // wait for splash to be well under way
```

Comment:
Flutter doesn't adhere to the 80 chars per line rule. So you can put this code on the same line as above.

Reply:
Done.